README.md

```diff
@@ -9,8 +9,10 @@ This test suite is designed to be executed against current versions of Red Hat's
 | UBI 8                    | 8               | [openjdk-8](https://catalog.redhat.com/software/containers/ubi8/openjdk-8/5dd6a48dbed8bd164a09589a)                                  |
 | UBI 8                    | 11              | [openjdk-11](https://catalog.redhat.com/software/containers/ubi8/openjdk-11/5dd6a4b45a13461646f677f4)                                |
 | UBI 8                    | 17              | [openjdk-17](https://catalog.redhat.com/software/containers/ubi8/openjdk-17/618bdbf34ae3739687568813)                                |
+| UBI 8                    | 21              | TBD                                                                                                                                  | 
```

Comment:
we've since shipped these images so you could fill out the table with the JDK21 image URIs now. Or do it in a future commit

Reply:
Future commit. At this point I just want to get this code in so we are ready for the CPU.